haskell-style.md

```diff
@@ -660,3 +660,329 @@ module Driver
     ) where
 
   ```
+
+## Haddocks
+
+Haskell modules should be commented with valid Haddock documentation. We are not
+yet requiring a certain level of coverage, but it is strongly encouraged.
+
+### General guides
+
+- Use proper Haddock [markup][]
```

Comment:
Is this meant to be a link to https://www.haskell.org/haddock/doc/html/index.html

Reply:
Oh wait, that is defined below. How have I never seen this link style in markdown?

Reply:
haha, I had the exact same train of thought you have here.